tracking/protocol.cpp

```diff
@@ -23,17 +49,56 @@ vector<uint8_t> Protocol::CreateAuthPacket(string const & clientId)
   return packet;
 }
 
-vector<uint8_t> Protocol::CreateDataPacket(DataElements const & points)
+vector<uint8_t> Protocol::CreateDataPacket(DataElementsCirc const & points)
```

Comment:
Yes, we need both of them. I think the same about iterators, but now i don't see solid reasons for refactoring it.